h/models/notification.py

```diff
@@ -20,6 +20,7 @@ class EmailTag(StrEnum):
     RESET_PASSWORD = "reset_password"  # noqa: S105
     MENTION_NOTIFICATION = "mention_notification"
     TEST = "test"
+    MODERATED = "moderated_notification"
```

Comment:
New email tag.

Reply:
The rest of the entries seem to be nouns. Maybe this one should be `MODERATION` instead of `MODERATED`?